docs/tutorials/send_receive_ingress_pem.md

```diff
@@ -275,19 +281,37 @@ EOF
 
 Wait for the Broker to be ready:
 
-```console
-$ kubectl get ActivemqArtemis  --watch
-NAME           READY   AGE
-send-receive   True    51s
+```{"stage":"deploy"}
+$ kubectl wait ActiveMQArtemis send-receive --for=condition=Ready --namespace=send-receive-project --timeout=120s
+activemqartemis.broker.amq.io/send-receive condition met
 ```
 
 Check that the ingress is available and has an IP address:
 
-```console
+```
 $ kubectl get ingress --show-labels
 NAME                                 CLASS   HOSTS                                                                     ADDRESS         PORTS     AGE   LABELS
 send-receive-sslacceptor-0-svc-ing   nginx   ing.sslacceptor.send-receive-0.send-receive-project.demo.artemiscloud.io  192.168.39.68   80, 443   18s   ActiveMQArtemis=send-receive,application=send-receive-app,statefulset.kubernetes.io/pod-name=send-receive-ss-0
+```
+
+#### Populate /etc/hosts
+
+Get the url for the ingress
 
+```{"stage":"etc", "variable":"INGRESS_URL", "runtime":"bash", "label":"get the ingress host"}
+$ kubectl get ingress send-receive-sslacceptor-0-svc-ing -o json | jq -r '.spec.rules[] | .host'
```

Comment:
The INGRESS_URL, this seems a little odd... should that variable not be part of the md command?

$ export INGRESS_URL=$(get ingress send....)

same or CLUSTER_IP?

Reply:
for when manually working through the tutorial